zebrad/src/components/sync/downloads.rs

```diff
@@ -309,7 +309,7 @@ where
 
                 if let Some(block_height) = block.coinbase_height() {
                     if block_height > max_lookahead_height {
-                        tracing::info!(
+                        debug!(
```

Comment:
I'd like to keep this log at info level, because it has helped us uncover bugs on fast machines or fast networks. It also helps the user configure their lookahead limit.

We should also add a hint about increasing the lookahead limit config to this log message. (Look for other "Hint:" logs for examples.)